### PR TITLE
Mapping Mapped: Willow models + contributing file

### PR DESCRIPTION
**Pull request by alinamstanciu**

### CODE REVIEW

**Review by joebeernink — APPROVED** [review]

File: OntologyMapper/src/Mappings/Mapped/Json/v0/Willow/mapped_json_v0_dtdlv2_willow.json [hunk 1]
```diff
@@ -22,346 +22,325 @@
...
       "InputDtmi": "dtmi:mapped:core:AirFlowRateCommand;1",
-      // Invalid Output DTMI - Review Needed
-      "OutputDtmi": "dtmi:com:willowinc:Air_Flow_Rate_Command;1",
-      "IsIgnored": true
+      "OutputDtmi": "dtmi:com:willowinc:Actuator;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:AirFlowSensor;1",
-      "OutputDtmi": "dtmi:com:willowinc:Air_Flow_Sensor;1"
+      "OutputDtmi": "tmi:com:willowinc:AirFlowSensor;1"
```
Comment (joebeernink) [inline OntologyMapper/src/Mappings/Mapped/Json/v0/Willow/mapped_json_v0_dtdlv2_willow.json:138]: typo: missing the d in dtmi. Caused test to fail

Comment (alinamstanciu) [reply OntologyMapper/src/Mappings/Mapped/Json/v0/Willow/mapped_json_v0_dtdlv2_willow.json:138]: Done

File: OntologyMapper/src/Mappings/Mapped/Json/v0/Willow/mapped_json_v0_dtdlv2_willow.json [hunk 2]
```diff
@@ -22,346 +22,325 @@
...
-      // Mapped to Base - Review needed
-      "OutputDtmi": "dtmi:com:willowinc:Equipment;1"
+      "OutputDtmi": "dtmi:com:willowinc:BACnetController;1"
     },
     {
       "InputDtmi": "dtmi:mapped:core:BACnetObjectId;1",
-      // Invalid Output DTMI - Review Needed
-      "OutputDtmi": "dtmi:com:willowinc:BACnet_Object_Id;1",
+      "OutputDtmi": "dtmi:com:willowinc:BACnetObjectId;1",
+      //"communication": "BACnet.ObjectId",
```
Comment (Tyler-Angell) [inline OntologyMapper/src/Mappings/Mapped/Json/v0/Willow/mapped_json_v0_dtdlv2_willow.json:308]: What is this comment?

Comment (alinamstanciu) [reply OntologyMapper/src/Mappings/Mapped/Json/v0/Willow/mapped_json_v0_dtdlv2_willow.json]: I've removed the left-over comment,

**Pull request merged**